src/styles/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  width: 200px;
```

Comment:
Use variables for values ​​wich may change over time, rather than replacing them everywhere manually